conf/webpack.config.js

```diff
@@ -31,7 +31,13 @@ module.exports = {
   devtool: 'source-map',
   module: {
     loaders: [
-      { test: /\.jsx$/, loader: "jsx-loader?harmony" }
+      {
+        test: /\.jsx$/,
+        loader: "babel",
+        query: {
+          presets: ['react']
```

Comment:
babel-loader should pick up .babelrc automatically so hopefully you don't need this.